spec/controllers/session_controller_spec.rb

```diff
@@ -27,19 +13,41 @@
     end
 
     it "logs in an existing user" do
-      @request.env["omniauth.auth"] = {
-        'provider' => 'twitter',
-        'info' => {'name' => 'Bob Jones',
-                   'nickname' => 'bjones'},
-        'uid' => '654321',
-        'extra' => {'raw_info' => {'profile_image_url_https' => 'http://robohash.org/1.png'}}
-      }
-      user = create(:user, uid: "654321")
+      @request.env["omniauth.auth"] = auth_data
+
+      user = create(:user, uid: "123456")
 
       post :create
 
       expect(User.count).to eq(1)
-      expect(User.first.uid).to eq("654321")
+      expect(User.first.uid).to eq("123456")
     end
   end
+
+  describe "development user" do
+    it "logs in a user" do
+      user = create(:user, name: "Bart Simpson", uid: "1")
+
+      post :create, "uid" => user.uid
+
+      expect(session["user_id"]).to eq User.last.id
+    end
+  end
+
+  def auth_data
+    {
+      'provider' => 'twitter',
+      'info' =>
+        {
+          'name' => 'Alice Smith',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.